src/utils/path.spec.ts

```diff
@@ -0,0 +1,9 @@
+import { removeLeadingSlashes } from './path';
+
+describe('removeLeadingSlashes', () => {
+  it('should remove leading slashes from a path', () => {
+    expect(removeLeadingSlashes('/test')).toEqual('test');
+    expect(removeLeadingSlashes('//test')).toEqual('test');
```

Comment:
This test won't pass currently. Please update the original files you submitted:

- src/utils/path.spec.ts
- src/utils/path.ts